### ISSUE TITLE
Unable to get started with a basic collector.

### ISSUE THREAD

**Issue by freecellwiz** [Hygieia/Hygieia#2604]
I'm running this locally on Windows 10 and do not have access to Gitter at work.  I have built everything but cannot get a collector working at all.  What I've done:

1) First I tried running just the api with  java -jar api.jar --spring.config.location="..\..\dashboard.properties" and the UI with gulp serve --local true.  I saw the dashboard but there was no data and I could not create any users.  I saw some earlier issues suggesting that the mock data model may be broken so I tried to configure a GitHub (Enterprise) collector.

2) I created application.properties for GitHub and tried to run the collector from the \target folder like this:
java -jar github-scm-collector-3.0.1.jar --spring.config.name=github --spring.config.location=c:\code\hygieia\collectors\scm\github\application.properties

I get an error: "Exception authenticating MongoCredential ... userName='dashboarduser', source='dashboarddb' ..."

I confirmed that MongoDB is running on the default port and that the dashboarddb and dashboarduser exist.  Password is just "test123" and is set as such in properties file.  Questions:

- what should I try next?
- once I fix this error, should I manually start the collector, api, UI in that order?  

Thanks!

**Comment by Sbrenthughes** [comment]
Can you post your properties file?

**Comment by freecellwiz** [comment]
Sure.  I'm probably missing something obvious.  
[application.properties.txt](https://github.com/Hygieia/Hygieia/files/2621172/application.properties.txt)

**Comment by Sbrenthughes** [comment]
Can you verify whether or not you are able to create a new user via the ui?

**Comment by Sbrenthughes** [comment]
It does not matter. I just wanted to make sure it was setup from the ui/api side. Since this is local, can you comment out the db user and db password and re run the collector?

**Comment by freecellwiz** [comment]
To clarify, when I ran with --local true what happened is that user creation failed.  It would just return me to the signup screen then on try #2 tell me the user existed.  I found issues where other people had this happen but didn't see an answer.  If I can get a real collector running I don't really care about the test data :-)

**Comment by Sbrenthughes** [comment]
Are you getting the mongo error while running the GitHub collector?

**Comment by freecellwiz** [comment]
Yes, the Mongo error is while running the collector.  I tried a few things and made some progress:

* When I run the GitHub collector I get "Exception authenticating MongoCredential{mechanism=SCRAM-SHA-1, userName='dashboarduser', source='dashboarddb', password=" 
* If I run the collector with the db user/pwd commented out in application.properties, it runs (no repo data is pulled yet but it's hitting GitHub - I can solve that part)
* I also commented out the db user/pwd in dashboard.properties for consistency, but now the login screen is missing the form.  

A bit later today (I'm at work) I may just trying deleting and creating the database in case I've gotten something in a bad state somehow.  I would like to understand the user/pwd issue.
